### PR TITLE
add file gitignore and extend webapp

### PR DESCRIPTION
**Pull request by hexoren21**

### CODE REVIEW

**Review by gitcollaboratorpk — APPROVED** [review]

**Review by gitcollaboratorpk — APPROVED** [review]

**Review by gitcollaboratorpk — APPROVED** [review]
m

**Review by gitcollaboratorpk — APPROVED** [review]
done

**Comment by hexoren21** [comment]
.

**Review by gitcollaboratorpk — APPROVED** [review]

File: main.tf [hunk 1]
```diff
@@ -25,6 +32,25 @@ resource "aws_security_group" "instance" {
...
+    	to_port     = 0
+    	protocol    = "-1"
+    	cidr_blocks = ["0.0.0.0/0"]
+	}
+}
+
+output "public_ip" {
+	value = aws_instance.example.public_ip
+	description = "Publiczny adres IP serwera WWW"
 }
```
Comment (gitcollaboratorpk) [inline main.tf:56]: ds

**Review by gitcollaboratorpk — APPROVED** [review]

**Review by gitcollaboratorpk — APPROVED** [review]

**Pull request merged**